package/MDAnalysis/analysis/helix_analysis.py

```diff
@@ -377,10 +376,7 @@ def __init__(self, universe, select='name CA', ref_axis=[0, 0, 1],
             counter = 0
             if len(groups) > 1:
                 msg = 'Your selection {} has gaps in the residues.'.format(s)
-                if split_residue_sequences:
-                    msg += ' Splitting into {} helices.'.format(len(groups))
-                else:
-                    groups = [ag.resindices]
+                groups = [ag.resindices]
```

Comment:
@lilyminium @fiona-naughton please review - I assume there was intent behind this?

Reply:
The tests fail because of this change.

Reply:
I checked the code again and the `groups` definition makes sense here. Sorry for the confusion. I added a proper doc entry.